rfc#0001

```diff
@@ -0,0 +1,140 @@
+# WASM binary loading into an Enarx Keep
+- Authors: [Mike Bursell](https://github.com/MikeCamel)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-01-10
+- Status Note: (has been subject to initial discussion and consensus)  
+- Supersedes: n/a
+- Start Date: 2020-02-16
+- Tags: architecture, webassembly
+
+## Summary
+
+This RFC describes the architecture adopted by Enarx for loading WASM
+packages into a prepared Keep. 
+
+## Motivation
+
+Once a Keep is prepared with microkernel, WASM runtime, etc., the
+workload (WASM binary + data) needs to be loaded into it.  A
```

Comment:
might we want to say "tenant workload" here to be very specific?

Reply:
Added.